### PR TITLE
 focused: boolean

### PR DESCRIPTION
**Pull request by jfcieslak**
`focused` property exists and is accessible on view instance.
see: https://github.com/ProseMirror/prosemirror-view/blob/master/src/index.js:34
It is not mentioned in docs, but this prop is actually the only reliable way to query whether view is focused. `hasFocus()` doesn't seem to work so well.

### CODE REVIEW

File: types/prosemirror-view/index.d.ts [hunk 1]
```diff
@@ -180,6 +180,8 @@ export class EditorView {
    * Focus the editor.
    */
   focus(): void;
+
+  focused: boolean;
```
Comment (bradleyayers) [inline types/prosemirror-view/index.d.ts]: > It is not mentioned in docs, but this prop is actually the only

This means that isn’t a private API and should not be relied on. As such it’s not appropriate to add to the types. 

If you’d like this API made public please raise an issue on prosemirror/prosemirror.

**Review by bradleyayers — CHANGES_REQUESTED** [review]

File: types/prosemirror-view/index.d.ts (hunk 1 above)
Comment (bradleyayers) [reply types/prosemirror-view/index.d.ts]: Isn’t a public API**

Comment (jfcieslak) [reply types/prosemirror-view/index.d.ts]: Makes sense, thanks!

**Comment by bradleyayers** [comment]
The other change is a dupe of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23242, so I think this can be closed for now.

**Comment by sheetalkamat** [comment]
Closing pull request as per feedback

**Pull request closed without merging**